app/plugins/datadog.md

```diff
@@ -55,11 +54,39 @@ parameter                      | default     | description
 Plugin currently logs following metrics to the Datadog server.
 
 Metric                     | description | namespace
----                        | ---         | -----
-`request_count`              | Increment the count of request made to the API by 1 | kong.\<api_name>.request.count
-`request_size`               | logs the request's body size in bytes | kong.\<api_name>.request.size
-`response_size`               | logs the response's body size in bytes | kong.\<api_name>.request.size
-`latency`                   | logs the time interval between the request started and response received from the upstream server | kong.\<api_name>.latency
-`status_count`               | For each status code returned, increment its counter by 1 | kong.\<api_name>.\<http_status_code>.count
-`unique_users`               | count of users made a request to the api | kong.\<api_name>.user.uniques
-`request_per_user`            | For each request by the user, increment its counter by 1 | kong.\<api_name>.\<consumer_id>.count
+---                        | ---         | ---
+`request_count`            | tracks api request | kong.\<api_name>.request.count
+`request_size`             | tracks api request's body size in bytes | kong.\<api_name>.request.size
+`response_size`            | tracks api response's body size in bytes | kong.\<api_name>.response.size
+`latency`                  | tracks the time interval between the request started and response received from the upstream server | kong.\<api_name>.latency
+`status_count`             | tracks each status code returned as response | kong.\<api_name>.status.\<status>.count and kong.\<api_name>.status.\<status>.total
+`unique_users`             | tracks unique users made a request to the api | kong.\<api_name>.user.uniques
+`request_per_user`         | tracks request/user | kong.\<api_name>.user.\<consumer_id>.count
+`upstream_latency`         | tracks the time it took for the final service to process the request | kong.\<api_name>.upstream_latency
+`kong_latency`             | tracks the internal Kong latency that it took to run all the plugins | kong.\<api_name>.kong_latency
+`status_count_per_user`    | tracks request/status/user | kong.\<api_name>.user.\<customer_id>.status.\<status> and kong.\<api_name>.user.\<customer_id>.status.total
+
+### Metric fields
+
+Plugin can be configured with any combination of list of [Metrics](#metrics), each entry consist of following fileds. 
```

Comment:
Sentence should read: "Plugin can be configured `with any combination of` [Metrics](#metrics), each entry `consists` of `the` following `fields`."

If the fields below are **required**, then the end of the sentence should read: "each entry `must contain` the following fields."